build-config/conf/busybox.config

```diff
@@ -727,7 +776,7 @@ CONFIG_WATCHDOG=y
 # CONFIG_SYS_EEPROM_SYSFS_FILE is not set
 CONFIG_SYS_EEPROM_I2C_DEVICE=""
 CONFIG_SYS_EEPROM_I2C_ADDR=
-CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=8
+CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=0
```

Comment:
OK, I think this will be fine.  When a system actually configures CONFIG_SYS_EEPROM this will be set correctly.